src/server.rs

```diff
@@ -975,24 +976,24 @@ mod tests {
             workspace_folders: None,
         };
 
-        let result = block_on(server.initialize(params)).unwrap();
+        let result = server.initialize(params).await.unwrap();
         let server_info = result.server_info.unwrap();
 
         assert_eq!(server_info.name, "flux-lsp".to_string());
         assert_eq!(server_info.version, Some("2.0".to_string()));
     }
 
     #[test]
-    fn test_shutdown() {
+    async fn test_shutdown() {
         let server = create_server();
 
-        let result = block_on(server.shutdown()).unwrap();
+        let result = server.shutdown().await.unwrap();
 
         assert_eq!((), result)
     }
 
     #[test]
-    fn test_did_open() {
+    async fn test_did_open() {
```

Comment:
There was a reason why we _didn't_ make these tests `async` before, but I don't remember what it was. Tests seem to be passing, so YOLO.